{
    "cveId": "CVE-2019-9948",
    "version": "1.0.0",
    "timestamp": "2024-12-12T22:08:52.294043+00:00",
    "description": "urllib in Python 2.x through 2.7.16 supports the local_file scheme, which makes it easier for remote attackers to bypass protection mechanisms that blacklist file URIs, as demonstrated by triggering a urllib.urlopen(local_file///etc/passwd) call.",
    "keyphrases": {
        "rootcause": "",
        "weakness": "",
        "impact": "",
        "vector": "local_file scheme",
        "attacker": "remote attackers",
        "product": "urllib",
        "version": "2.x through 2.7.16",
        "component": ""
    }
}
